### PR TITLE
Technical Blog "Build a Twitter Bot Using NodeJS"

### PR DESCRIPTION
**Pull request by Ninjavin**
#### Please note that in case of below checklist not updated accordingly, the maintainers have the right to immediately close your pull request.

- [x] You created a branch from `dev` branch.
- [x] You PR is raised on `dev` branch and not on`master`.
- [x] Blog follows [Blog Guidelines](https://github.com/LoginRadius/engineering-portal/blob/master/GUIDELINES.md).
- [x] You have read the [Contribution Guidlines](https://github.com/LoginRadius/engineering-portal/blob/master/CONTRIBUTING.md) before creating this PR.
- [x] Checked for any duplicate blog entry.
- [x] Checked for all grammatical errors in the blog.
- [x] You have not published the blog content anywhere else on the internet on platforms like medium, dev.to etc.
- [x] The content is not copy-pasted from anywhere else on the Internet.

##### Author Details for Blog Post Entry

id: Vineeta Jain
bio: Third-year undergraduate and a Full Stack Web Developer. A team-player with a can-do attitude.
github: [Ninjavin](https://github.com/Ninjavin)
twitter: [vineeta_vj](https://twitter.com/vineeta_vj)
medium: [vinjain2006](https://medium.com/@vinjain2006)

### CODE REVIEW

**Review by puneetsingh24 — COMMENTED** [review]
@Ninjavin As per our contribution guidelines, can you please resize the cover image to 1200px to 800px

**Comment by Ninjavin** [comment]
@puneetsingh24 I have resized the image. Thank you.

**Review by puneetsingh24 — CHANGES_REQUESTED** [review]
Hello @Ninjavin 

Thanks for writing this blog and we appreciate your effort while writing this. However, as per our project contribution guidelines, we recommend uploading the source code used in the blog post to the repo - https://github.com/LoginRadius/engineering-blog-samples and give the link to the folder in the blog.

Can you please create a separate pull request on blog samples repo?

Thanks

**Comment by Ninjavin** [comment]
@puneetsingh24 I have created a separate PR on Engineering Blog Samples repo PR [#82](https://github.com/LoginRadius/engineering-blog-samples/pull/82).
Thank you.

File: content/blog/build-a-twitter-bot-using-nodejs/index.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+### 4. Start Coding 🎯
+
+I will use [Official Joke API](https://github.com/15Dkatz/official_joke_api) to automatically fetch content for my tweet. You can visit the repository and learn more about the endpoints. But the only [endpoint](https://github.com/15Dkatz/official_joke_api#grab-a-random-joke) we will work with is [random_joke](https://official-joke-api.appspot.com/random_joke). Click on this and you will receive a JSON response with fields id, type, setup and punchline. The fields we are concerned with are setup and punchline. 
+
+We will use [axios](https://github.com/axios/axios) for calling the API endpoint. Run `npm install axios` and add `var axios = require('axios');` in `index.js`.
+
+Now everything has been laid out. We need to connect the dots. The Official Joke API will return a JSON response. We will grab the required data i.e. the setup and the punchline from it. And then make use of [client.post](https://github.com/desmondmorris/node-twitter/tree/master/examples#tweet) functionality of **twitter** package.
+
+```js
+axios("https://official-joke-api.appspot.com/random_joke").then(Response => {
```
Comment (vyasriday) [inline content/blog/build-a-twitter-bot-using-nodejs/index.md:74]: @Ninjavin Does axios support api request like this? Shouldn't this be either a `.get` or a config object`?

**Review by vyasriday — APPROVED** [review]

**Review by puneetsingh24 — APPROVED** [review]

**Pull request merged**